### PR TITLE
Angie's stuff

### PR DESCRIPTION
**Pull request by angiephantastic**

### CODE REVIEW

**Comment by ugurtekbas** [comment]
@angiephantastic 
>Column size is responsive but not the lines of text.

It's great you noticed that, no worries we can make it responsive as well.

>Cookie buttons are not working.

It's totally fine it shouldn't be working

File: newsPaper.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              <button onclick="myFunction()" id="myBtn">Read more</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  <!--<SCRIPT TYPE="text/javascript"> function popup(mylink, windowname)
+    { if (! window.focus)return true; var href; if (typeof(mylink) == 'string')
+    href=mylink; else href=mylink.href; window.open(href, windowname, 'width=400,height=200,scrollbars=yes'); return false; }
+  </SCRIPT>
+  <BODY onLoad="popup('creeper.jpg', 'ad')">-->
```
Comment (ugurtekbas) [inline newsPaper.html]: Will you need this commented code later?

File: newsPaper.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+<div class="cookie-consent-banner" >
```
Comment (ugurtekbas) [inline newsPaper.html:84]: Please format your code.

**Review by ugurtekbas — CHANGES_REQUESTED** [review]
Please edit your description about the homework.

File: newsPaper.html (hunk 1 above)
Comment (angiephantastic) [reply newsPaper.html]: Maybe I do but I can remove it from here.

**Pull request merged**